SystemFiles/Government.cpp

```diff
@@ -1,51 +1,69 @@
 #include "Government.h"
 
-Government::Government() {
-	// TODO - implement Government::Government
-	throw "Not yet implemented";
+#include <iostream>
+
+Government::Government(int govermentBalance) {

```

Comment:
Why does this function take a parameter if it initializes the balance to 0? 

Also this is completely up to you but if you like you can just hard code it an initial balance of like 50,000.